src/main/java/com/google/devtools/build/lib/rules/android/BusyBoxActionBuilder.java

```diff
@@ -344,10 +344,12 @@ public void buildAndRegister(String message, String mnemonic) {
 
     if (dataContext.isPersistentBusyboxToolsEnabled()) {
       commandLine.add("--logWarnings=false");
-      spawnActionBuilder
-          .addCommandLine(commandLine.build(), WORKERS_FORCED_PARAM_FILE_INFO);
-
+      spawnActionBuilder.addCommandLine(commandLine.build(), WORKERS_FORCED_PARAM_FILE_INFO);
       executionInfo.putAll(ExecutionRequirements.WORKER_MODE_ENABLED);
+
+      if (dataContext.isPersistentMultiplexBusyboxTools()) {
+        executionInfo.putAll(ExecutionRequirements.WORKER_MULTIPLEX_MODE_ENABLED);
```

Comment:
So this flag would override `--noexperimental_worker_multiplex`?

Reply:
Passing `--noexperimental_worker_multiplex` would disable multiplex workers entirely while `isPersistentMultiplexBusyboxTools` just instructs `BusyBoxActionBuilder` to register it's actions with with `supports-multiplex-worker` specified as part of it's execution info.

Reply:
D'uh. You're right.